nyc_taxis/challenges/default.json

```diff
@@ -187,25 +236,18 @@
         {
           "operation": {
             "operation-type": "force-merge",
-            "request-timeout": 7200
+            "request-timeout": 7200{%- if force_merge_max_num_segments is defined %},
+            "max-num-segments": {{ force_merge_max_num_segments | tojson }}
+            {%- endif %}
           }
         },
         {
           "name": "refresh-after-force-merge",
           "operation": "refresh"
         },
         {
-          "name": "wait-until-merges-finish",
-          "operation": {
-            "operation-type": "index-stats",
-            "index": "_all",
-            "condition": {
-              "path": "_all.total.merges.current",
-              "expected-value": 0
-            },
-            "retry-until-success": true,
-            "include-in-reporting": false
-          }
+          "operation": "wait-until-merges-finish",
+          "include-in-reporting": false
```

Comment:
`include-in-reporting` is an operation parameter, not a task parameter so it needs to be defined on level deeper. This property also affects only the command line report; metrics are always pushed to the metrics store.